torchio/data/queue.py

```diff
@@ -146,15 +147,39 @@ def __init__(
         self.shuffle_subjects = shuffle_subjects
         self.shuffle_patches = shuffle_patches
         self.samples_per_volume = samples_per_volume
+        if type(samples_per_volume) == int:
```

Comment:
Please use `isinstance`, as in https://github.com/fepegar/torchio/blob/a51248ee52382bc639f45e46c98032a5ed33f33b/torchio/data/subject.py#L69